packages/packem/src/packem.ts

```
@@ -663,6 +666,9 @@ const packem = async (
     });
 
     let logged = false;
+    let onSuccessProcess: ExecChild | undefined;
+    // eslint-disable-next-line @typescript-eslint/no-invalid-void-type,@typescript-eslint/no-explicit-any
+    let onSuccessCleanup: (() => any) | undefined | void;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)